.github/workflows/stale.yml

```
@@ -3,8 +3,14 @@ on:
   schedule:
     - cron: "40 17 * * *"
 
+permissions:
+  contents: read
+
```

Given that the only job in this workflow is using a custom token, we should limit all scopes.

```suggestion
permissions: {}

```